base/intset.jl

```diff
@@ -143,7 +143,7 @@ function intersect!(s1::IntSet, s2::IntSet)
 end
 
 setdiff(s::IntSet, ns) = setdiff!(copy(s), ns)
-setdiff!(s::IntSet, ns) = (for n in ns; _delete!(s, n); end; s)
+setdiff!(s::IntSet, ns) = (for n in ns; delete!(s, n); end; s)
```

Comment:
Yes I think so, but it seems unavoidable since there's no way to know that everything in `ns` is safe in general.